### ISSUE TITLE
Update script seems to be not working

### ISSUE THREAD

**Issue by JohnQMetro** [mikeeq/mbp-fedora-kernel#26]
The download process does not seem to work properly, and even if the rpm files are downloaded manually, the script 'Can not load RPM file' for all of them.

**Comment by mikeeq** [comment]
Any outputs from the script?

**Comment by JohnQMetro** [comment]
[](`===]> Info: Updating update_kernel_mbp script... 
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  3555  100  3555    0     0  19532      0 --:--:-- --:--:-- --:--:-- 19426
===]> Info: Downloading latest kernel... 
===]> Info: Latest kernel version: 5.12.11 
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100     9  100     9    0     0     26      0 --:--:-- --:--:-- --:--:--    26
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100     9  100     9    0     0     17      0 --:--:-- --:--:-- --:--:--    17
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100     9  100     9    0     0     25      0 --:--:-- --:--:-- --:--:--    25
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100     9  100     9    0     0      7      0  0:00:01  0:00:01 --:--:--     7
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100     9  100     9    0     0     26      0 --:--:-- --:--:-- --:--:--    26
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100     9  100     9    0     0     37      0 --:--:-- --:--:-- --:--:--    37
100     9  100     9    0     0     37      0 --:--:-- --:--:-- --:--:--    37
Invalid configuration value: failovermethod=priority in /etc/yum.repos.d/fedora-updates-modular.repo; Configuration: OptionBinding with id "failovermethod" does not exist
Invalid configuration value: failovermethod=priority in /etc/yum.repos.d/fedora-updates-modular.repo; Configuration: OptionBinding with id "failovermethod" does not exist
Invalid configuration value: failovermethod=priority in /etc/yum.repos.d/fedora-updates-modular.repo; Configuration: OptionBinding with id "failovermethod" does not exist
Last metadata expiration check: 0:01:40 ago on Tue 29 Jun 2021 01:56:12 PM.
Can not load RPM file: ./kernel-5.12.11-300.mbp.fc33.x86_64.rpm.
Can not load RPM file: ./kernel-core-5.12.11-300.mbp.fc33.x86_64.rpm.
Can not load RPM file: ./kernel-devel-5.12.11-300.mbp.fc33.x86_64.rpm.
Can not load RPM file: ./kernel-modules-5.12.11-300.mbp.fc33.x86_64.rpm.
Can not load RPM file: ./kernel-modules-extra-5.12.11-300.mbp.fc33.x86_64.rpm.
Can not load RPM file: ./kernel-modules-internal-5.12.11-300.mbp.fc33.x86_64.rpm.
Could not open: ./kernel-5.12.11-300.mbp.fc33.x86_64.rpm ./kernel-core-5.12.11-300.mbp.fc33.x86_64.rpm ./kernel-devel-5.12.11-300.mbp.fc33.x86_64.rpm ./kernel-modules-5.12.11-300.mbp.fc33.x86_64.rpm ./kernel-modules-extra-5.12.11-300.mbp.fc33.x86_64.rpm ./kernel-modules-internal-5.12.11-300.mbp.fc33.x86_64.rpm
`
I'll note that the 'Invalid configuration value' lines did not appear the first time I tried the script.  I also tried to upgrade from Fedora 32 to Fedora 33, but that did not work very well (keyboard did not work).
